migrations/001.do.init.sql

```diff
@@ -22,4 +22,23 @@ CREATE TABLE hunt (
     enemy_id INT NOT NULL references enemies(id) ON DELETE CASCADE,
     PRIMARY KEY(character_id, enemy_id)
 );
-ALTER TABLE players ADD COLUMN active_character INT REFERENCES characters(id) ON DELETE SET NULL;
+CREATE TYPE enum_material AS ENUM('Dragon Talon', 'Dragon Hide', 'Dragon Fang');
+CREATE TABLE materials (
+    type enum_material NOT NULL,
+    quantity (int),
+    PRIMARY KEY(player_id, material_id)
+);
```

Comment:
You are missing some columns here - the primary key can only reference columns that exist.

Also `quantity (int)` is not a valid thing...

```suggestion
CREATE TABLE materials (
    player_id INT NOT NULL REFERENCES players (id) ON DELETE CASCADE,
    type enum_material NOT NULL,
    quantity INT NOT NULL DEFAULT 0,
    PRIMARY KEY(player_id, type)
);
```

Reply:
I forgot what was going through my head during that time. I'll change it to `quantity INT`.